models/goosefx/solana/goosefx_ssl_v2_solana_trades.sql

```diff
@@ -0,0 +1,144 @@
+ {{
+  config(
+        schema = 'goosefx_ssl_v2_solana',
+        alias = 'trades',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_id', 'outer_instruction_index', 'inner_instruction_index', 'tx_index','block_month']
+        )
+}}
+
+{% set project_start_date = '2023-08-01' %} --grabbed program deployed at time (account created at).
+
+  WITH
+    pools as (
+        --technically not used below, but here for a dex_solana.pools spell later on.
+        SELECT 
+            tkA.symbol as tokenA_symbol
+            , tkA.decimals as tokenA_decimals
+            , ip.account_mintOne as tokenA
+            , ip.account_mintOneSecondaryVault as tokenAVault
+            , tkB.symbol as tokenB_symbol
+            , tkB.decimals as tokenB_decimals
+            , ip.account_mintTwo as tokenB
+            , ip.account_mintTwoSecondaryVault as tokenBVault
+            , cast(null as varchar) as fee_tier
+            , ip.account_pair as pool_id
+            , ip.call_tx_id as init_tx
+            , ip.call_block_time as init_time
+        FROM {{ source('goosefx_solana', 'gfx_ssl_v2_call_createPair') }} ip 
+        LEFT JOIN {{ ref('tokens_solana_fungible') }}  tkA ON tkA.token_mint_address = ip.account_mintOne
+        LEFT JOIN {{ ref('tokens_solana_fungible') }}  tkB ON tkB.token_mint_address = ip.account_mintTwo
+    )
+
+    , all_swaps as (
+        SELECT 
+            sp.call_block_time as block_time
+            , 'goosefx_ssl' as project
+            , 2 as version
```

Comment:
We may use `'2'` as varchar typed version.
```suggestion
            , '2' as version
```

Reply:
I used integers for all versions across dex_solana trades for now, so we can't do this